lua/lv-which-key/init.lua

```diff
@@ -74,10 +71,24 @@ vim.api.nvim_set_keymap('n', '<leader>p', ":lua require'telescope'.extensions.pr
 local mappings = {
     ["/"] = "Comment",
     ["c"] = "Close Buffer",
-    ["e"] = "Explorer",
+    ["e"] = {"<cmd>lua require\"lv-nvimtree\".toggle_tree()<CR>", "Explorer"},
     ["f"] = "Find File",
     ["h"] = "No Highlight",
     ["p"] = "Projects",
+    ['<TAB>']   =  {'<cmd>BufferNext<cr>', 'jump to next buffer'},
+    ['<S-TAB>'] =  {'<cmd>BufferPrevious<cr>' , 'jump to prev buffer'},
```

Comment:
I would also argue that these two are not necessary since already present in [keymappings.lua](https://github.com/ChristianChiarulli/LunarVim/blob/master/lua/keymappings.lua)

Reply:
Those are not `barbar` commands btw, and I'm also surprised that we have yet another dupe 😞 

My idea was to add some extra functionality and not really refactor anything in particular. So now the latest patch contains only a minimal change that provides some missing keybindings, while not really bothering with unifying/moving anything else.